examples/compiler/00_index.ipynb

```diff
@@ -0,0 +1,53 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Devito's Operator \n",
+    "\n",
+    "In Devito, an `Operator` carries out three fundamental tasks: generation of low-level code, JIT compilation, and execution. Overall, the lowering process performed by an `Operator` – from high-level equations to dynamically compiled and executable code – consists of multiple compiler passes, summarized in the Figure bellow. \n",
+    "\n",
+    "![Compiler Stages](https://i.ibb.co/dKjsZ72/compiler-stages.png)\n",
+    "\n",
+    "The following series of notebooks will expose key aspects of those compilation stages in a hands-on fashion. We will be exploring the heart of the `Operator` class, which can be found at `devito/operator.py`.\n",
+    "\n",
+    "The examples and exercices shown here are mostly taken either from tests found in the folder `devito/tests`, or from the documentation itself.\n",
```

Comment:
exercises

Reply:
I think we can actualy safely drop this sentence, or at least be more conservative ("Many of the examples shown in these notebooks are extracted from...") but I'd not say all of them or even mostly.

Reply:
thank you @georgebisbas , fixed.

ok @FabioLuporini fixed.